src/main/java/io/xpring/xrpl/DefaultXRPClient.java

```diff
@@ -292,6 +292,28 @@ public boolean accountExists(String address) throws XRPException {
     }
   }
 
+  @Override
+  /**
+   * Retrieve the transaction corresponding to the given transaction hash.
+   * <p>
+   * Note: This method can return transactions that are not included in a fully validated ledger.
+   *       See the `validated` field to make this distinction.
+   * </p>
+   * @param transactionHash The hash of the transaction to retrieve.
+   * @return An XRPTransaction object representing an XRP Ledger transaction.
+   * @throws XRPException If the transaction hash was invalid.
+   */
+  public XRPTransaction getTransaction(String transactionHash) throws XRPException {
```

Comment:
I think this will only convert payment objects right? 

Maybe rename as `getPayment`?